.storybook/main.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
         actions: false
       }
     },
-    "@storybook/addon-a11y"
+    "@storybook/addon-a11y",
+    "@storybook/addon-controls"
```

Comment:
Removing controls should solve this but doing it this way also swapped the order where the Accessibility tab is first by default instead of Controls.

Reply:
I'm sorry, I don't quite understand the the importance of the order. Can you please explain?

Reply:
Actually, I think I know what you mean now. In the situation where we're not able to do what I need without the controls addon, would swapping the order here change which tab is first?

Reply:
Yep the order here matters for which tab is displayed first in the addons pannel. Controls was before Accessibility because it is first in the addons array.

Reply:
Also when a user lands on a story they should see the Controls addon open by default in the panel, since that is more important and what we're likely to use first compared to Accessibility. Hope that clarifies what I meant by the first comment.